docs/managing-dragonfly/flags.md

```diff
@@ -0,0 +1,535 @@
+# Command line arguments (flags)
+
+Dragonfly can be tuned and configured by a set of config flags. These flags can be:
+
+1. Passed to a Dragonfly as command line arguments. E.g, `./dragonfly --port=6379`.
```

Comment:
```suggestion
1. Passed as command line arguments. E.g, `./dragonfly --port=6379`.
```

Reply:
Also, I wonder if we should drop the `./` part of running Dragonfly, as most users won't run it from the local directory?